src/js/events/ChronosActions.js

```diff
@@ -133,6 +134,26 @@ const ChronosActions = {
     });
   },
 
+  updateJob: function (jobID, data) {
```

Comment:
I have the same disadvantage like you `@aldipower` my interpretation of the jobs would be that update is right because you update a timer but you edit the time? Jobs are running in a ir-/regular timeframe Service do run constantly. That is my interpretation but it doesn't mean I am right.